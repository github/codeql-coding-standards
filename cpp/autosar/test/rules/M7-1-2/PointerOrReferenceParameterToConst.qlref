rules/M7-1-2/PointerOrReferenceParameterToConst.ql