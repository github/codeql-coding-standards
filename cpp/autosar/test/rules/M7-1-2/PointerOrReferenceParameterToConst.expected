| test.cpp:4:13:4:14 | p2 | Parameter p2 points to an object that is not modified and therefore should be declared as const. |
| test.cpp:5:19:5:20 | p3 | Parameter p3 points to an object that is not modified and therefore should be declared as const. |
| test.cpp:19:12:19:12 | a | Parameter a points to an object that is not modified and therefore should be declared as const. |
| test.cpp:50:12:50:12 | a | Parameter a points to an object that is not modified and therefore should be declared as const. |
